chia/wallet/puzzles/p2_delegated_puzzle_or_hidden_puzzle.py

```diff
@@ -73,8 +73,6 @@
 
 MOD = load_clvm("p2_delegated_puzzle_or_hidden_puzzle.clvm")
 
-SYNTHETIC_MOD = load_clvm("calculate_synthetic_public_key.clvm")
```

Comment:
do we have any clvm code that depends on this now? or is this part of a library of useful chialisp functions? `calculate_synthetic_public_key.clvm`

Reply:
Dont think so, it can be removed. Not sure why we did it in clvm in the first place